### DISCUSSION TITLE
It will be the best CMS ever. If microfeed develops other than Podcasts, try developing it again for the Blog feature. 😁

### DISCUSSION THREAD

**Discussion by ridvanmaulana** [microfeed/microfeed#89] [Show and tell]
Hello Microfeed!

I've tried changing the default theme of the microfeed. So like this https://microfeed.ridvanmau.com/
![microfeeddd](https://github.com/microfeed/microfeed/assets/12792021/7d8cfc4d-2a87-4eba-9ff4-7d126ff51081)

Microfeeds are amazing. Indeed, we know that microfeed can be connected to other static generators such as Hugo, etc.

But for me it sucks. Because there is not much freedom. In other words, it's just a "post".

Meanwhile, microfeed is amazing. Not all CMS can be self hosted with Cloudflare. Frontend and backend work together.

Maybe microfeed can add features like those of a standard blog CMS such as tags, menus, posts/pages. at least 3, that's all, microfeed will definitely be the king of CMS because of its convenience. 😁😁😁

Success Microfeed. Hope it continues to grow!

**Comment by orawalters** [comment]
All due respect, but why would it add the features of a standard blog CMS? In my opinion, the source of Microfeed's simplicity is that it doesn't try to replicate the features of CMSs that already exist but instead appeals to people who just want a small feed  (Microfeed, if you will) without all the distracting bells and whistles.

I'm not a representative of this project, and I'm not saying yes/no to any specific requests, just clarifying that I think some of the missing components were actually intentional.

**Reply by ridvanmaulana** [reply]
Wow, I just realized and understand what microfeed means. thanks for letting me know.. 😁

Yes, it looks like the simplified feature is intentional. And focus as file storage with feeds. But I still hope that just one of the microfeeds gives one feature "tags" lol hahaha 😁

Microfeed remains the cms of the future. And forefront. Because the first one is using Pages, 1 and R2 for hosts..

thanks for your attention!

**Reply by marycamacho** [reply]
Agreed - I know it is lightweight - but it suggests that the uses are podcast, blogs, photos, documents and videos. The features are very good for podcasts (I use it for my podcast.) and they are less good for blogs (I also use it for my blog.)

I would love to see tags added for blogs, URLs, photos, documents and videos... as well as podcasts. These other uses are not limited by Apple and Spotify syndication, and a bit more flexibility would make all those more functional when using a feed to display in another context. 

For me the extreme value of microfeed is the fabulous documentation, easy setup and that in one day, you can own your feeds on your custom domain. That has become near imposible in the age of platforms.
